File: 백준/Bronze/30018. 타슈/타슈.cc
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    int n,rlt=0;
    cin>>n;
    vector<int> a(n),b(n);
    for(int i=0;i<n;i++)
        cin>>a[i];
    for(int i=0;i<n;i++) {
        cin>>b[i];
        rlt+=abs(a[i]-b[i]);
    }
    cout<<rlt/2;
    return 0;
}

File: 백준/Bronze/30018. 타슈/타슈.py
import sys
input=lambda:sys.stdin.readline().rstrip()
n,rlt=int(input()),0
a=[*map(int,input().split())]
b=[*map(int,input().split())]
for i in range(n):
    rlt+=abs(a[i]-b[i])
print(rlt//2)